server/controllers/cohort.js

```diff
@@ -122,7 +130,7 @@ const updateCohortById = (req, resp) => {
         return resp.sendStatus(400);
       }
 
-      resp.send();
+      cohort = doc;
```

Comment:
you don have to do it like this.
You can use helper function 
```
returnPayload(socketActions.updateCohort(
          socketInstance,
          allCohortsViewClients
        )(data))(doc);
```
and in the last `.then` you will get this doc as payload 
```

.then(doc/payload => { 
 fireAndForget(
        saveActivity(
          cohortActivity,
          userId,
          null,
          null,
          sanitizedCohortId,
          null,
          payload.name
        )
      );	      );
}
```

Please refactor this in all places